gst/droidcamsrc/gstdroidcamsrc.c

```diff
@@ -3,6 +3,7 @@
  *
  * Copyright (C) 2014 Mohammed Sameer <msameer@foolab.org>
  * Copyright (C) 2015-2016 Jolla LTD.
+ * Copyright (C) 2010 Texas Instruments, Inc
```

Comment:
Where did this come from? It's in the header too.

Reply:
Sorry, forget to mention this.

The code for handling the preview pipeline is pretty much copied from `gstbasecamerasrc` in gst-plugins-bad [1]. You'll see that the code is copyrighted by TI and licensed under LGPL2+. Thus, re-distribute with LGPL2.1+ should is compatible.

[1] https://gitlab.freedesktop.org/gstreamer/gst-plugins-bad/-/blob/master/gst-libs/gst/basecamerabinsrc/gstbasecamerasrc.c

Reply:
Ok, useful piece of information there, thanks! Aligning with the base impl is a useful target.